packages/ui/src/components/NumberInput/NumberInput.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { lgStyleProps, mdStyleProps, NumberInput, smStyleProps } from './NumberInput';
+
+describe('NumberInput', () => {
+  it('renders correctly in not disabled mode', () => {
+    render(<NumberInput size="lg" inputMode="numeric" disabled={false} aria-label="inputNumber" />);
```

Comment:
Don't add props that wouldn't affect the test to the component. In this case, only the `aria-label` prop is needed for this test to work properly